components/sensor/sht3xd/sht3xd_nonlinux.go

```diff
@@ -1,3 +1,2 @@
-// This is blank for non linux
-
+// Package sht3xd This is blank for non linux
```

Comment:
This comment was causing lint errors on macOS so I added the correct comment format to fix.